### DISCUSSION TITLE
DAQmx error

### DISCUSSION THREAD

**Discussion by qqspm** [bonsai-rx/bonsai#1624] [Q&A]
![Capture](https://github.com/bonsai-rx/bonsai/assets/26908946/12d3d95b-bb08-4e12-8a38-e556faccf3fb)

Trying to use DigitalInput (USB-6501) as a trigger for audio playing, but I got an error like this.
Please help!!!

**Comment by bruno-f-cruz** [comment]
Heya, 

Happy to help, but you will need to provide a little bit more information...How do you want to use a DigitalInput to trigger a audio stimuli? You might want to look at the package documentation (https://bonsai-rx.org/daqmx/articles/intro.html).

I would also urge you to read the guidelines for posting in the discussion group (https://github.com/orgs/bonsai-rx/discussions/864). Try to use a bit more descriptive titles that might help other users in the future. 

Thanks!

**Reply by qqspm** [reply]
![image](https://github.com/bonsai-rx/bonsai/assets/26908946/26603121-1bbf-4973-93f4-352105979188)
![image](https://github.com/bonsai-rx/bonsai/assets/26908946/2bd262bc-4a7d-42b0-a38b-e49d774281c2)
![image](https://github.com/bonsai-rx/bonsai/assets/26908946/50b07d64-b4db-4822-a8b8-b6ac86fcf9b7)

Here is the setting I am using. 
Thank you for the help!

**Reply by bruno-f-cruz** [reply]
Please provide the workflow file as instructed in #864.

**Comment by qqspm** [comment]
[5khz.zip](https://github.com/bonsai-rx/bonsai/files/13342123/5khz.zip)

Attached is the workflow and the wav file to be played.
I am using USB-6501 as the device. Driver is up to date.
Thank you!

**Reply by bruno-f-cruz** [reply]
Again, i don't understand what you are trying to achieve in your experiment, are you trying to trigger the sound on some sort of logic switch (High -> Low? Low to high?), it is just really hard to provide assistance if you dont provide details of your experiment... Any timing constraints?

**Reply by qqspm** [reply]
I am trying to use the Digital input to trigger a sound playing. We have a device to deliver different TTL out, which connects to this NI device as DigitalInput to trigger a sound delivery, from low to high. As long as there is a TTL in, the sound will be played. Not specific timing constraints.

**Reply by qqspm** [reply]
This is a part of our two choice experiment. Different DigitalInput channel to trigger different sound to be played.

**Reply by qqspm** [reply]
The problems is even just simply using the DigitalInput only without any other component, the error comes as the first screenshot,
